app/controllers/application_controller.rb

```diff
@@ -625,7 +625,7 @@ def calculate_lastaction(lastaction)
   end
 
   def report_edit_aborted(lastaction)
-    add_flash(_("Edit aborted!  ManageIQ does not support the browser's back button or access from multiple tabs or windows of the same browser.  Please close any duplicate sessions before proceeding."), :error)
+    add_flash(_("Edit aborted!  %s does not support the browser's back button or access from multiple tabs or windows of the same browser.  Please close any duplicate sessions before proceeding.") % I18n.t('product.name'), :error)
```

Comment:
All the `%s` in this pull request should be replaced by a nice placeholder, for example `%{product_name}`. Not a big deal, it's just something that makes the life of the translators easier.